src/components/FormComponents/SourceWizardSummary.js

```diff
@@ -89,6 +112,11 @@ export const createItem = (formField, values, stepKeys) => {
     return undefined;
   }
 
+  // do not show fields hidden just for review step
+  if (formField.hideInReview) {
+    return undefined;
```

Comment:
`undefined` can't be rendered. Wouldn't it be better to return `null`?
```suggestion
    return null;
```

Reply:
we are still applying `filter(Boolean)` at the function's output. Changed the whole function to return `null` everywhere to make it safer

Reply:
@karelhala it does not probably make sense. The output of the function is not meant to be rendered. 

`filter(Boolean)` is applied to the output and then the results are mapped again into components there:
```
const valuesList = valuesInfo.map(({ label, value }) => (
    <DesctiptionListItem
      key={`${label}--${value}`}
      term={label}
      description={value.toString().length > 150 ? <ValuePopover label={label} value={value} /> : value}
    />
  ));
```